internal/events/events.go

```diff
@@ -126,14 +126,25 @@ func eventString(writer io.Writer, event flow.Event) {
 }
 
 func printField(writer io.Writer, field cadence.Field, value cadence.Value) {
+	v := value.String()
 	var typeId string
+
+	defer func() {
+		if err := recover(); err != nil {
+			v = fmt.Sprintf("%s\n\t\thex: %x", v, v)
+			_, _ = fmt.Fprintf(writer, "\t\t- %s (%s): %s \n", field.Identifier, "?", v)
```

Comment:
extract this print into a method so it's not duplicated

Reply:
addressed